samples/Hardware.GiantGecko/category.txt

```diff
@@ -0,0 +1 @@
+ggecko
```

Comment:
So a new category created, and need to be reflected in the build tool ;-)

Reply:
Absolutely! And it's there and listed on the readme's.